src/main/webapp/filterScript.js

```diff
@@ -0,0 +1,43 @@
+const UNCLICKED = 'unclicked-filter';
+const CLICKED = 'clicked-filter';
+const CLICKED_ARRAY_URL = 'clicked-array=';
+
+/**
+* Turn a filter a different color when pressed, and change class accordingly.
+**/
+function turnBlueWhenClicked() { //eslint-disable-line
+  const allFilters = document.querySelectorAll('.unclicked-filter');
+  for (let i = 0; i < allFilters.length; i++) {
+    allFilters[i].onclick = function() {
+      const currFilter = allFilters[i];
+      if (currFilter.classList.contains(UNCLICKED)) {
+        currFilter.classList.remove(UNCLICKED);
+        currFilter.classList.add(CLICKED);
+      } else {
+        currFilter.classList.remove(CLICKED);
+        currFilter.classList.add(UNCLICKED);
+      }
+    };
```

Comment:
No need to change it, but if you haven't seen [arrow functions](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Functions/Arrow_functions) in javascript before, let me introduce you to them! Arrow functions are nice because they produce shorter code, and also handle the `this` binding differently. If you're not familiar with javascript scope and the `this` keyword, you can read [this article](https://medium.com/tfogo/advantages-and-pitfalls-of-arrow-functions-a16f0835799e#:~:text=Arrow%20functions%20are%20best%20for,of%20this%20in%20their%20scope.).

Reply:
I have seen arrow functions! I think I'm just more comfortable with this syntax since it's more like declaring a function how I would normally, but I can change it to arrow syntax if it's better in this case